w-books/src/config/components/PrivateRoute/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
```

Comment:
Great! just a detail, This arent components, are routes... So i think that this is better.
´´´
config
   |  routes.js
   |  PrivateRoute.js
   |  PublicRoute.js
```

All as simple files into config folder